xds/src/test/java/io/grpc/xds/XdsClientWrapperForServerSdsTestMisc.java

```diff
@@ -51,12 +57,14 @@
   @Mock private Channel channel;
 
   private XdsClientWrapperForServerSds xdsClientWrapperForServerSds;
+  private XdsClient.ListenerWatcher registeredWatcher;
 
   @Before
   public void setUp() throws IOException {
     MockitoAnnotations.initMocks(this);
     xdsClientWrapperForServerSds = new XdsClientWrapperForServerSds(PORT);
-    xdsClientWrapperForServerSds.start(xdsClient);
+    registeredWatcher =
+        XdsServerBuilderTest.startAndGetWatcher(xdsClientWrapperForServerSds, xdsClient, PORT);
```

Comment:
Might be better to move static helper methods to a test util class.

Reply:
moved to a new helper test class.